src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -205,6 +223,18 @@ protected void writeToNBT(NBTTagCompound compound)
         }
         compound.setTag(TAG_CITIZENS, citizenTagList);
 
+        //  Achievements
+        NBTTagList achievementsTagList = new NBTTagList();
+        for (Achievement achievement : this.colonyAchievements)
+        {
+            //todo: get the key from the achievement
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1135)